BLSQ/iaso
# POLIO-858: Polio / Budget: migrate repeat button to steps

Whimsical [link](https://whimsical.com/move-repeat-step-button-8JFFU8XomPLPhKfoxp9fUu)

Moving repeat step button form the top of the page to the correct row of step in step table

Related JIRA tickets : POLIO-858

## Self proofreading checklist


## Changes

- removed the option to display a repeat action from `AddStepButton` and add `RepeatStepIconProps`
- pre-filter repeat steps in `BudgetDetailsInfos` and pass it to steps table
- Create an action cell for steps
- display repeat button on steps with a repeat action possible

## How to test

Open an existing budget for a specific campaign and country.
Proceed to steps.
You should see the loop icon (mainly on the last step)
Click on it and re submit the step

## Print screen / video

https://user-images.githubusercontent.com/12494624/223716910-5118356e-7ade-4721-97ce-4bfd901fe4d8.mov


## Notes

I'm wondering if we should not display that a step has been repeated in the table
